CSharpMath.Avalonia.Example/MainView.xaml.cs

```diff
@@ -8,8 +8,8 @@ public class MainView : UserControl {
     public MainView() {
       InitializeComponent();
 
-      var light = (Styles)AvaloniaXamlLoader.Load(new System.Uri("avares://Avalonia.Themes.Fluent/Accents/FluentLight.xaml"));
-      var dark = (Styles)AvaloniaXamlLoader.Load(new System.Uri("avares://Avalonia.Themes.Fluent/Accents/FluentDark.xaml"));
+      var light = (Styles)AvaloniaXamlLoader.Load(new System.Uri("avares://Avalonia.Themes.Fluent/FluentLight.xaml"));
+      var dark = (Styles)AvaloniaXamlLoader.Load(new System.Uri("avares://Avalonia.Themes.Fluent/FluentDark.xaml"));

```

Comment:
```suggestion
      var dark = new FluentTheme() { Mode = FluentThemeMode.Dark};

```

Reply:
I not find a constructor without arguments for FluentTheme, it may only works from xaml code I think.
@jp2masa the compiler accepts that constructor using 0.10.0 ?

Reply:
Sorry, you're right, I think you can pass `null`.

Reply:
np, thx for suggestion about on App.xaml and for info about avalonia 0.10.0 released.